### PR TITLE
Benchmarks

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+using Microsoft.AspNetCore.Razor.Evolution;
+
+namespace Microsoft.AspNetCore.Razor.Performance
+{
+    [Config(typeof(CoreConfig))]
+    public class CodeGenerationBenchmark
+    {
+        public CodeGenerationBenchmark()
+        {
+            var current = new DirectoryInfo(Environment.CurrentDirectory);
```
Comment (davidfowl) [inline test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs]: Use `[Setup]` for this?

File: test/Microsoft.AspNetCore.Razor.Performance/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Compile the solution in Release mode (so binaries are available in release)
+
+To run a specific benchmark add it as parameter
+```
+dotnet run -c Release <benchmark_name>
+```
+To run all use `All` as parameter
```
Comment (pakrym) [inline test/Microsoft.AspNetCore.Razor.Performance/readme.md]: @davidfowl is this true with new `BenchmarkSwitcher`?

Comment (davidfowl) [inline test/Microsoft.AspNetCore.Razor.Performance/readme.md]: Nope. We should nuke this from all of our repos.

File: test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs (hunk 1 above)
Comment (davidfowl) [inline test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs]: Hmm, maybe not.

Comment (rynowak) [inline test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs]: What's the difference?

Comment (rynowak) [inline test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs]: I couldn't find a good way to deal with external files so I'm just doing this.

File: test/Microsoft.AspNetCore.Razor.Performance/readme.md (hunk 2 above)
Comment (rynowak) [inline test/Microsoft.AspNetCore.Razor.Performance/readme.md]: OK I'll delete this from the readme.

File: test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs (hunk 1 above)
Comment (davidfowl) [inline test/Microsoft.AspNetCore.Razor.Performance/CodeGenerationBenchmark.cs]: Yea leave it

**Pull request closed without merging**